### ISSUE TITLE
Initial WindowPlacement.Fullscreen doesn't work on macOs

### ISSUE THREAD

**Issue by kyoybs** [JetBrains/compose-jb#1134]
I find the codes in "tutorials/Window_API outdated", but it's outdated.

**Comment by igordmn** [comment]
This should work:
```
import androidx.compose.ui.window.Window
import androidx.compose.ui.window.WindowPlacement
import androidx.compose.ui.window.application
import androidx.compose.ui.window.rememberWindowState

fun main() = application {
    val state = rememberWindowState(placement = WindowPlacement.Fullscreen)

    Window(onCloseRequest = ::exitApplication, state) {
        
    }
}
```

**Comment by igordmn** [comment]
Checked it on macOs, it doesn't work. It is a bug.

**Comment by igordmn** [comment]
Should be fixed in 1.0.0-rc3

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-6207">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
